diff --git a/sections/speakers/panel_appreciate.pug b/sections/speakers/panel_appreciate.pug
index dd045f5..e43f367 100644
--- a/sections/speakers/panel_appreciate.pug
+++ b/sections/speakers/panel_appreciate.pug
@@ -3,21 +3,21 @@
 br
 .container.speaker-list
   .col-lg-3.col-xs-6.speaker-item
-    a.speaker-link(href='michael-duff', data-toggle='modal')
+    a.speaker-link(href='#michael-duff', data-toggle='modal')
       .speaker-hover
         .speaker-hover-content
       img.img-responsive(src='images/speaker/mc2018/michaelduff.png', alt='')
     .speaker-caption
       h4 Michael Duff
   .col-lg-3.col-xs-6.speaker-item
-    a.speaker-link(href='carlo-contaldi', data-toggle='modal')
+    a.speaker-link(href='#carlo-contaldi', data-toggle='modal')
       .speaker-hover
         .speaker-hover-content
       img.img-responsive(src='images/speaker/mc2018/carlocontaldi.png', alt='')
     .speaker-caption
       h4 Carlo Contaldi
   .col-lg-3.col-xs-6.speaker-item
-    a.speaker-link(href='veras', data-toggle='modal')
+    a.speaker-link(href='#veras', data-toggle='modal')
       .speaker-hover
         .speaker-hover-content
       img.img-responsive(src='images/speaker/mc2018/veras.png', alt='')
